static/templates/actions_popover_content.hbs

```diff
@@ -27,21 +28,27 @@
     </li>
     {{/if}}
 
-    {{#if should_display_quote_and_reply}}
+    {{#if should_display_delete_option}}
     <li>
-        <a class="respond_button" data-message-id="{{message_id}}" tabindex="0">
-            <i class="fa fa-reply" aria-hidden="true"></i> {{t "Quote and reply or forward" }}
-            <span class="hotkey-hint">(>)</span>
+        <a class="delete_message" data-message-id="{{message_id}}" tabindex="0">
+            <i class="fa fa-times" aria-hidden="true"></i>
+            {{t "Delete message" }}
         </a>
     </li>
     {{/if}}
 
-    {{#if view_source_menu_item}}
+    {{#if (or editability_menu_item move_message_menu_item should_display_delete_option)}}
```

Comment:
Cannot find any better way to do this. We can also compute it in javascript but yeah this seems atleast better than that.